fix(docs-infra): add swap to web fonts so downloading does not block…

ajitsinghkaler opened the pull request:
… rendering

Light house was reporting that 'Ensure text remains visible during webfont load' solution to this was adding & swap to the end o web fonts this leds to our first text showing before web-font download and improves the performance of out site link to article: https://web.dev/font-display/\?utm_source\=lighthouse\&utm_medium\=lr

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
Web fonts block rendering. 

Issue Number: N/A


## What is the new behavior?
Web fonts won't block rendering while downloading.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

mary-poppins commented:
You can preview 4d879e9 at https://pr35352-4d879e9.ngbuilds.io/.

gkalpak commented on aio/src/index.html:
```diff
@@ -20,9 +20,9 @@
-  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Droid+Sans+Mono&display=swap">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons&display=swap">
```
Since this is an icon font, I don't think it makes sense to show a fallback font.

gkalpak reviewed (changes requested):
Thx, @ajitsinghkaler :+1:
Please change the commit message to have `fix` as scope: `fix(docs-infra): ...`

mary-poppins commented:
You can preview 26a452a at https://pr35352-26a452a.ngbuilds.io/.

ajitsinghkaler commented:
@gkalpak made the changes please have a look

gkalpak commented on aio/src/index.html:
```diff
@@ -20,8 +20,8 @@
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Droid+Sans+Mono">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Droid+Sans+Mono&display=swap">
   <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```
Actually, based on [this thread][1] (and mentioned resources), I think `display: block` is the appropriate value for icon fonts (which will also make LightHouse happy):

```suggestion
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons&display=block">
```

[1]: https://github.com/GoogleChrome/lighthouse/issues/10127

ajitsinghkaler committed:
fix(docs-infra): add fallback to web fonts so downloading does not block rendering

Light house was reporting that 'Ensure text remains visible during webfont load' solution to this problem was adding &swap to the end of web fonts this leads to our first text showing before web-font download and improves the performance of site link to article: https://web.dev/font-display/\?utm_source\=lighthouse\&utm_medium\=lr

mary-poppins commented:
You can preview 5ba991f at https://pr35352-5ba991f.ngbuilds.io/.

gkalpak reviewed (approved):
👌
